processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -210,7 +209,7 @@ public ImmutableBitmap getBitmapIndex(String dimension, String value)
 
     final Column column = index.getColumn(dimension);
     if (column == null || !columnSupportsFiltering(column)) {
-      if (Strings.isNullOrEmpty(value)) {
+      if (value == null) {
```

Comment:
would this still full fill the smooth transition path without checking `NullHandlingHelper.isNullOrDefault`